src/main/java/com/google/songgame/servlets/UserServlet.java

```diff
@@ -49,9 +53,43 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(userEntity);
   }
 
+  @Override
+  public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Map<String, String> roomProperties = readJSONFromRequest(request);
+
+    Query userQuery =
+        new Query("User").addFilter("userId", FilterOperator.EQUAL, roomProperties.get("userId"));
+    PreparedQuery result = datastore.prepare(userQuery);
+    Entity currentUser = result.asSingleEntity();
```

Comment:
Currently the room seems to store only one user. Is the userId only for admins? And other users aren't stored in the rooms? If so can you change the property into something more generic, like adminUserId?

Reply:
@salnad The user is stored in the room, but the current room is stored for each user one at a time as they join the room

Reply:
Ah alright, I think I was misunderstanding this. Thought it was RoomServlet not UserServlet. Looks good!